esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -1427,39 +1452,22 @@ void EsdlServiceImpl::prepareFinalRequest(IEspContext &context,
     // access to them, so pull them from ourself. When they are passed in the
     // error checking has already been completed.
```

Comment:
This comment is no longer accurate since the CRTs are no longer passed in. 
N.B. I also intend to verify that this refactoring continues to do the error checking required in all cases.